src/snapshot.zig

```diff
@@ -0,0 +1,159 @@
+const std = @import("std");
+const base = @import("base.zig");
+const parse = @import("check/parse.zig");
+
+var verbose_log: bool = false;
```

Comment:
nit: I think zig has a builtin param to set the log level that we could use instead of this. Then just enable log level info or debug to get these prints.

Reply:
I tried that and couldn't get it working.